src/jobs/activities/wallet-activity-tracking.ts

```diff
@@ -32,6 +34,39 @@ export const walletTransactionLogsQueue = new Queue(
   queueOptions
 );
 
+const getCacheKey = (accountId: string, txHash: string) => `${accountId}:${txHash}`;
+
+const invokeWebhookEndpoints = async (
+  transferEvent: TransferEventData,
+  payload: WalletActivityEvent
+) => {
+  const txHash = transferEvent.transactionHash;
+
+  const accountTransactionCache = new Map<string, WebhookRequest>();
```

Comment:
This cache works only while processing each transfer event. So that same user_transaction (tx hash) is not sent to each service multiple times